src/index.js

```diff
@@ -119,6 +119,18 @@ class RCTVoice {
       });
     });
   }
+
+  /**
+   * (Android) Get a list of the speech recognition engines available on the device
+   * */
+  getSpeechRecognitionServices() {
+    if (Platform.OS !== 'android') {
```

Comment:
can we just return?

this would require more code on user side

or return an empty array

Reply:
it makes sense, tks

Reply:
Thanks. :)